File: ApiSuvesaPos/Negocio/Modelo/cxc/ajustesccobrar/ajustesccobrar.vb
Namespace Modelo
    Public Class ajustesccobrar
        Public Property ID_Ajuste As Long
        Public Property AjusteNo As Long
        Public Property Tipo As Char
        Public Property Cod_Cliente As Long
        Public Property Nombre_Cliente As String
        Public Property Fecha As DateTime
        Public Property Saldo_Prev As Decimal
        Public Property Monto As Decimal
        Public Property Saldo_Act As Decimal
        Public Property Observaciones As String
        Public Property Anula As Boolean
        Public Property Cod_Usuario As String
        Public Property Contabilizado As Boolean
        Public Property Cod_Moneda As Integer
        Public Property Asiento As Long
        Public Property Detalle_Ajustesccobrar As New List(Of detalle_ajustesccobrar)
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/ajustesccobrar/ajustesccobrar.cs
namespace Modelo
{
    public class ajustesccobrar
    {
        public long ID_Ajuste { get; set; }
        public long AjusteNo { get; set; }
        public char Tipo { get; set; }
        public long Cod_Cliente { get; set; }
        public string Nombre_Cliente { get; set; }
        //public DateTime Fecha { get; set; }
        public decimal Saldo_Prev { get; set; }
        public decimal Monto { get; set; }
        public decimal Saldo_Act { get; set; }
        public string Observaciones { get; set; }
        public bool Anula { get; set; }
        public string Cod_Usuario { get; set; }
        public bool Contabilizado { get; set; }
        public int Cod_Moneda { get; set; }
        public long Asiento { get; set; }
        //public List<detalle_ajustesccobrar> Detalle_Ajustesccobrar { get; set; } = new List<detalle_ajustesccobrar>();
    }
}


